bundles/org.openhab.binding.dbquery/README.md

```diff
@@ -0,0 +1,213 @@
+
+    {
+        correct : true,
+        data : [
+            { 
+                column1 : value,
+                column2 : value
+            },
+            { ... }, //row2        
+            { ... }  //row3
+        ]
+    }
+    
+### Channel Triggers
+
+#### calculateParameters
+
+Triggers when there's a need to calculate parameters before query execution.
+When a query has `hasParameters=true` it fires the `calculateParameters` channel trigger and pause the execution until `setQueryParameters` action is call in
+ that query.
+ 
+In the case a query has parameters it's expected that there is some rule that catch the `calculateParameters` trigger, calculate the parameters
+ with the corresponding logic and then call the `setQueryParameters` action, after that the query will be really executed.
+ 
+## Actions
+
+### For DatabaseBridge
+
+#### executeQuery 
+
+It allows executing a query synchronously from a script/rule without defining it in a Thing.
+
+To execute the action you need to pass the following parameters:
+
+- String query: The query to execute
+- Map<String,Object>: Query parameters (empty map if not needed)
+- int timeout: Query timeout in seconds
+
+And it returns an `ActionQueryResult` that has the following properties:
+
+- correct (boolean) : True if the query was executed correctly, false otherwise
+- data (List<Map<String,Object>>): A list where each element is a row that is stored in a map with (columnName,value) entries  
+- isScalarResult: It returns if the result is scalar one (only one row with one column)
+- resultAsScalar: It returns the result as a scalar if possible, if not returns null
+
+
+Example (using Jython script):
+
+     from core.log import logging, LOG_PREFIX 
+     log = logging.getLogger("{}.action_example".format(LOG_PREFIX))
+     map = {"time" : "-2h"}
+     influxdb = actions.get("dbquery","dbquery:influxdb2:sampleQuery") //Get bridge thing
+     result = influxdb.executeQuery("from(bucket: \"default\") |> range(start:-2h)  |> filter(fn: (r) => r[\"_measurement\"] == \"go_memstats_frees_total\")  |> filter(fn: (r) => r[\"_field\"] == \"counter\")  |> mean()",{},5)
+     log.info("execute query result is "+str(result.data))
+    
+
+Use this action with care, because as the query is executed synchronously is not good to execute long-running queries and block script execution.
+
+### For Queries
+
+#### setQueryParameters
+
+It's used for queries with parameters to set them. To execute the action you need to pass the parameters as a Map.
+
+Example (using Jython script):
+
+    params = {"time" : "-2h"}
+    dbquery = actions.get("dbquery","dbquery:query:queryWithParams")  //Get query thing
+    dbquery.setQueryParameters(params)
+
+#### getLastQueryResult
+
+It can be used in scripts to get the last query result. It doesn't have any parameters and returns an `ActionQueryResult` as defined in `executeQuery` action.
```

Comment:
```suggestion
It can be used in scripts to get the last query result.
It doesn't have any parameters and returns an `ActionQueryResult` as defined in `executeQuery` action.
```